Use proper resource name in sample code

nathenharvey opened the pull request:
the control should describe an `aws_ec2` resource, not an `ec2` resource.

Signed-off-by: Nathen Harvey <nharvey@chef.io>

nathenharvey committed:
Use proper resource name in sample code

the control should describe an `aws_ec2` resource, not an `ec2` resource.

Stiffni committed:
Add to_s method to aws_iam_user (#60)

Signed-off-by: sfreeman <Steffanie.Freeman@d2l.com>

mtjandra committed:
Prompt for namespace if not set in environment variable instead of providing a random namespace (#57)

Signed-off-by: Miles Tjandrawidjaja <miles@tjandrawidjaja.com>

chris-rock commented:
@nathenharvey Not sure why the DCO bot went south. Its seems like this commit is not properly signed-off.

Stiffni committed:
Add to_s method to aws_iam_password_policy (#61)

* Add to_s method to aws_iam_password_policy

Signed-off-by: sfreeman <Steffanie.Freeman@d2l.com>

* Use single quoted string and remove unnecessary substring

Signed-off-by: sfreeman <Steffanie.Freeman@d2l.com>

mtjandra committed:
Ensure resources are cleaned when running test:integration even if integration tests fail (#56)

Signed-off-by: Miles Tjandrawidjaja <miles@tjandrawidjaja.com>

Stiffni committed:
Run linter on unit tests (#51)

* Run linter on unit tests

Signed-off-by: sfreeman <Steffanie.Freeman@d2l.com>

* Bring remaining unit tests up to spec with rubocop

Signed-off-by: sfreeman <Steffanie.Freeman@d2l.com>

chris-redekop committed:
A real-world working AwsIamUsers (#71)

* Add aws_iam_users

Signed-off-by: Chris Redekop <chris.redekop@d2l.com>

* Adding Filter table and Collect User Details to aws_iam_users.rb

Signed-off-by: Chris Redekop <chris.redekop@d2l.com>

* Adding Filter table and Collect User Details to aws_iam_users.rb

Signed-off-by: Chris Redekop <chris.redekop@d2l.com>

* Adding Filter table and Collect User Details to aws_iam_users.rb

Signed-off-by: Chris Redekop <chris.redekop@d2l.com>

* Get an aws_iam_users integration test to pass

Signed-off-by: Chris Redekop <chris.redekop@d2l.com>

* Fix RuboCop issues and tests

Signed-off-by: Chris Redekop <chris.redekop@d2l.com>

* Improving code based on PR feedback

Signed-off-by: Chris Redekop <chris.redekop@d2l.com>

chris-redekop committed:
Add has_mfa_enabled? to aws_iam_root_account (#80)

Signed-off-by: Chris Redekop <chris.redekop@d2l.com>

chris-rock committed:
update readme with available resources (#84)

Signed-off-by: Christoph Hartmann <chris@lollyrock.com>

jerryaldrichiii committed:
Change `ec2(` to `aws_ec2(` in README.md (#87)

Signed-off-by: jerry <jerry@chef.io>

nathenharvey committed:
README code examples are copy-paste-able. (#86)

The resource is `aws_ec2`, not `ec2`

Instance IDs must be passed to the `aws_ec2` resource, not machine names

Signed-off-by: Nathen Harvey <nharvey@chef.io>

jerryaldrichiii committed:
Add support for `ENV['AWS_REGION']` (#92)

The `aws-sdk` will raise an error mentioning `ENV['AWS_REGION']` if you
omit `ENV['AWS_DEFAULT_REGION']`. This adds support for either via `||`.

Signed-off-by: Jerry Aldrich <jerryaldrichiii@gmail.com>

clintoncwolfe committed:
Update Terraform commands for v0.10 (#93)

* Terraform now requires init to fetch providers

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Rename env terraform command to workspace

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

* Pin Terraform version to reflect CLI updates

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

chris-rock committed:
rename file for aws_ec2 resource

* rename to aws_ec2_instance

Signed-off-by: Christoph Hartmann <chris@lollyrock.com>

clintoncwolfe committed:
aws_ec2 reource has been renamed to aws_ec2_instance

Signed-off-by: Clinton Wolfe <clintoncwolfe@gmail.com>

clintoncwolfe commented:
Apologies, should have rebased on that last push.  

On the good side, I seem to have tickled the DCO bot into being happy again.

chris-rock commented:
@nathenharvey I apologize for the late response. We decided to go with #97 instead. Please let me know if we missed something.
